src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -21,6 +21,11 @@ import {
     getTextWidth,
     truncateWithEllipsis,
 } from 'cbioportal-frontend-commons';
+import { GENERIC_ASSAY_CONFIG } from 'shared/lib/GenericAssayUtils/GenericAssayConfig';
```

Comment:
If there is any way we can keep MiniScatter agnostic of generic assay stuff, that would probably be best.  but i see that line was crossed before.

Reply:
Maybe I should create another component for Generic Assay?